src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -727,12 +699,10 @@ private JComboBox getJComboBoxMedical() {
 					Medical med = ((Medical) jComboBoxMedical.getSelectedItem());
 					jButtonChooseLot.setEnabled(false);
 					jButtonNewLot.setEnabled(true);
-					lotExist = false;
 					for (MedicalWard medWard : wardDrugs) {
 						if (med.getDescription().equals(medWard.getMedical().getDescription())) {
 							jButtonChooseLot.setEnabled(true);
 							jButtonNewLot.setEnabled(false);
-							lotExist = true;
```

Comment:
I've doubts concerning this removal... maybe we lost some logic back in date?

Reply:
There are only three occurrences of `lotExist`; one is the definition and the other two are here.  The value is never tested so the assignment are not needed.   You added the code on June 12, 2020 and as far as I can tell looking at other versions between now and then nothing was ever done with the variable.

Reply:
I can see that, but I'm wandering if there was any logic before that time... maybe we need to verify that part: the JComboBox should show only medicals with a lot whose quantity is different from zero. 
Maybe this check is done elsewhere in the same class?

Reply:
I checked all 26 commits between now and when the code as added on June 12, 2020.  There are always 3 occurrences of `lotExist`.  The private definition of the variable; the set to false; and the set to true.

I then checked all 12 commits before June 12, 2020 and none of them had the variable.

It looks like it was an incomplete thought that is lost in time.....

Reply:
yes, could be! I will test it deeply... thanks!